src/Components/CollectionsHubRails/index.tsx

```diff
@@ -37,13 +35,12 @@ export const CollectionsHubRails = ({ marketingCollection }: Props) => {
 export const CollectionsHubRailsContainer = createFragmentContainer(
   CollectionsHubRails,
   {
-    marketingCollection: graphql`
-      fragment CollectionsHubRails_marketingCollection on MarketingCollection {
-        linkedCollections {
-          groupType
-          ...FeaturedCollectionsRails_collectionGroup
-          ...OtherCollectionsRail_collectionGroup
-        }
+    linkedCollections: graphql`
```

Comment:
Can you give me some context as to why and how this fragment changed? I don't have any complaints or suggestions, I just don't really understand it.

Reply:
Same! Would love to understand this better!

Reply:
Previously we were passing in the entire collection (i.e. `marketingCollection`). For the hubs rail we only need the `linkedCollection` which is a property on the collection so I changed the fragment to reflect that.

Does that makes sense?

Reply:
Totally, thanks for the clarification!